src/pages/guides/index.md

```diff
@@ -217,16 +207,19 @@ Note: While your event registration is marked `Disabled`, Adobe will continue to
 
 ## Security Considerations
 
-Your webhook URL must necessarily be accessible from the open internet. This means third-party actors can send forged requests to it, tricking your application into handling fake events.
+<InlineAlert variant="info" slots="text"/>
+Adobe strongly encourages securing your webhook deliveries to avoid processing malicious request.
 
-To prevent this from happening, Adobe I/O Events has a robust event validation process in place as defined below that allows users to secure their webhook.
+Indeed, as your webhook URL must necessarily be accessible from the open internet, 
+third-party actors can send forged requests to it, tricking your application into handling fake events.
 
-<InlineAlert variant="info" slots="text"/>
-Adobe strongly encourages validating your webhook deliveries using this new mechanism to avoid processing "events" received from malicious third-party actors and make sure your webhook continues to receive events.
+To prevent this from happening, Adobe I/O Events offers two options for securing your webhook deliveries:
+* Adobe I/O Events Digital Signature 
+* Adobe I/O Events  `mTLS` support
 
-### Improved and Resilient Security Verification for Webhook Events
+Both options are described below, pick just one, implementing both would be a waste of resources.
```

Comment:
im wondering if we should say wast of resources. does not sound very professional. 
may be rephrase it to : 

You should choose one of these options to implement. Using both is unnecessary and would only consume additional resources without added benefit.